lib/transfer.rb

```diff
@@ -0,0 +1,72 @@
+class Transfer
```

Comment:
@Gataniel put it in `app/models`. `lib` folder is for library files. But this is business logic.